probe-rs/targets/esp32p4.yaml

```
@@ -0,0 +1,44 @@
+name: esp32p4
+manufacturer:
+  cc: 0x0C
+  id: 0x12
+variants:
+  - name: esp32p4
+    part: null
+    jtag:
+      scan_chain:
+        - name: lpcpu
+          ir_len: 5
+        - name: hp
+          ir_len: 5
+    default_binary_format: idf
+    cores:
+      - name: main
+        type: riscv
+        core_access_options: 
+          !Riscv
+            jtag_tap: 1
```

```suggestion
            jtag_tap: 1
            hart_id: 0
```

While implicitly there already, it's a good example for the second core later